tanner/tests/test_rfi_emulation.py

```diff
@@ -19,8 +18,9 @@ def test_http_download_fail(self):
         path = 'http://foobarfvfd'
         filename = self.loop.run_until_complete(self.handler.download_file(path))
         self.assertIsNone(filename)
-
+        
     def test_ftp_download(self):
+        self.handler.download_file_ftp = mock.MagicMock(return_value = '10ddd27e6bab5c8e2aa356906d1b71e5')
         path = 'ftp://mirror.yandex.ru/archlinux/lastupdate'
         data = self.loop.run_until_complete(self.handler.download_file(path))
         self.assertIsNotNone(data)
```

Comment:
I suggest you assert with exactly returned value for mocked function. It checks that mocked function was called and proper value was used in download_file function